java/core/src/main/java/co/worklytics/psoxy/gateway/impl/CommonRequestHandler.java

```diff
@@ -430,11 +431,19 @@ HttpRequestFactory getRequestFactory(HttpEventRequest request) {
 
         ComposedHttpRequestInitializer initializer =
                 ComposedHttpRequestInitializer.of(initializeWithCredentials,
-                        new GzipedContentHttpRequestInitializer("Psoxy"));
+                        new GzipedContentHttpRequestInitializer(getProxyUserAgentString()));
```

Comment:
I don't think this does anything at all, as is used as fallback

https://github.com/Worklytics/psoxy/blob/331a45abe814ae046b5f37f8fb4907aa17b9d83d/java/core/src/main/java/co/worklytics/psoxy/utils/GzipedContentHttpRequestInitializer.java#L25C9-L26C39

so presumably we are sending this instead always:

https://github.com/googleapis/google-http-java-client/blob/f2e14545476c83ed36ef46839a00f2fa000acf75/google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java#L67

Also, should that class be  `Gzipped...` with double P?